src/Accordion/accordion.js

```diff
@@ -44,22 +44,18 @@ class Accordion extends Component {
             activeItems: activeItems,
         });
 
-        if (this.props.onChange) {
-            this.props.onChange(this.props.accordion ? activeItems[0] : activeItems);
-        }
+        this.props.onChange(this.props.accordion ? activeItems[0] : activeItems);
     }
 
     renderItems() {
         const { accordion, children } = this.props;
-        if (!children) {
-            return null;
-        }
 
         return React.Children.map(children, (item, index) => {
             const key = index;
             const expanded = (this.state.activeItems.indexOf(key) !== -1) && (!item.props.disabled);
```

Comment:
I'm not sure why you set `key = index` if they are identical – why not just use `index`? Unless `Children.map` has a special behavior from `Array#map` I do not understand.